packages/sitebuilder/src/contexts/GoLiveProvider.tsx

```diff
@@ -80,6 +81,7 @@ const GoLiveProvider = ({ children }: { children: React.ReactNode }) => {
 
 	const navigate = useNavigate();
 	const [searchParams, setSearchParams] = useSearchParams();
+	const nexcessNavigation = searchParams.get('nexcess') === 'true';
```

Comment:
There's been a request to change the query string variable `nexcess` to `purchase`.

Reply:
Done